packages/browser-policy-content/browser-policy-content.js

```diff
@@ -102,19 +102,26 @@ var addSourceForDirective = function (directive, src) {
   if (_.contains(_.values(keywords), src)) {
     cspSrcs[directive].push(src);
   } else {
-    src = src.toLowerCase();
-
-    // Trim trailing slashes.
-    src = src.replace(/\/+$/, '');
-
     var toAdd = [];
-    // If there is no protocol, add both http:// and https://.
-    if (! /^([a-z0-9.+-]+:)/.test(src)) {
-      toAdd.push("http://" + src);
-      toAdd.push("https://" + src);
+
+    //Only add single quotes to CSP2 script digests
+    if (/^((sha)(256|384|512)-)/.test(src)) {
```

Comment:
Ah, just an artifact - before I checked the w3c spec, I assumed other hashing algorithms would be allowed so I was building a more complex regex. 

Now that I'm looking at it more closely, it should be case insensitive, too: ```/^sha(256|384|512)-/i.test(src)```